File: Code/Quickstarts/cs/UIProcessQuickstarts_NoNavGraph/WinUI/congratulation.cs
//===============================================================================
// Microsoft User Interface Process Application Block for .NET Quick Start
// http://msdn.microsoft.com/library/en-us/dnbda/html/uip.asp
//
// congratulation.cs
//
// This file contains the implementations of the congratulation class.
//
// 
//===============================================================================
// Copyright (C) 2000-2001 Microsoft Corporation
// All rights reserved.
// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY
// OF ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT
// LIMITED TO THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR
// FITNESS FOR A PARTICULAR PURPOSE.
//==============================================================================

using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;

using Microsoft.ApplicationBlocks.UIProcess;

namespace UIProcessQuickstarts_Store.WinUI
{
	public class congratulation : WindowsFormView
	{
		private System.Windows.Forms.Button returnButton;
		private System.Windows.Forms.Label msgLabel;
        
		/// <summary>
		/// Required designer variable.
		/// </summary>
		private System.ComponentModel.Container components = null;

		public congratulation()
		{
			//
			// Required for Windows Form Designer support
			//
			InitializeComponent();
		}

		/// <summary>
		/// Clean up any resources being used.
		/// </summary>
		protected override void Dispose( bool disposing )
		{
			if( disposing )
			{
				if(components != null)
				{
					components.Dispose();
				}
			}
			base.Dispose( disposing );
		}

		#region Windows Form Designer generated code
		/// <summary>
		/// Required method for Designer support - do not modify
		/// the contents of this method with the code editor.
		/// </summary>
		private void InitializeComponent()
		{
            this.msgLabel = new System.Windows.Forms.Label();
            this.returnButton = new System.Windows.Forms.Button();
            this.SuspendLayout();
            // 
            // msgLabel
            // 
            this.msgLabel.Location = new System.Drawing.Point(32, 24);
            this.msgLabel.Name = "msgLabel";
            this.msgLabel.Size = new System.Drawing.Size(232, 32);
            this.msgLabel.TabIndex = 0;
            this.msgLabel.Text = "Congratulations. Your order has been sent sucessfully";
            this.msgLabel.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
            // 
            // returnButton
            // 
            this.returnButton.Location = new System.Drawing.Point(96, 72);
            this.returnButton.Name = "returnButton";
            this.returnButton.Size = new System.Drawing.Size(104, 24);
            this.returnButton.TabIndex = 1;
            this.returnButton.Text = "Return";
            this.returnButton.Click += new System.EventHandler(this.returnButton_Click);
            // 
            // congratulation
            // 
            this.AutoScaleBaseSize = new System.Drawing.Size(5, 13);
            this.ClientSize = new System.Drawing.Size(288, 109);
            this.ControlBox = false;
            this.Controls.Add(this.returnButton);
            this.Controls.Add(this.msgLabel);
            this.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedDialog;
            this.MaximizeBox = false;
            this.Name = "congratulation";
            this.StartPosition = System.Windows.Forms.FormStartPosition.CenterScreen;
            this.Text = "Congratulations";
            this.Load += new System.EventHandler(this.congratulation_Load);
            this.ResumeLayout(false);

        }
		#endregion

		private void returnButton_Click(object sender, System.EventArgs e)
		{
			StoreController.StartNewShoppingSession();
		}

        private void congratulation_Load(object sender, System.EventArgs e)
        {
        }

        #region UIPManager Plumbing
        private StoreController StoreController
        {
            get{ return (StoreController)this.Controller; }
        }
        #endregion
	}
}

File: Code/Quickstarts/vb/UIProcessQuickstarts_Store/WinUI/congratulation.vb
 '===============================================================================
' Microsoft User Interface Process Application Block for .NET Quick Start
' http://msdn.microsoft.com/library/en-us/dnbda/html/uip.asp
'
' congratulations.cs
'
' This file contains the implementations of the congratulations class.
'
' 
'===============================================================================
' Copyright (C) 2000-2001 Microsoft Corporation
' All rights reserved.
' THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY
' OF ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT
' LIMITED TO THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR
' FITNESS FOR A PARTICULAR PURPOSE.
'==============================================================================
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports System.Windows.Forms

Imports Microsoft.ApplicationBlocks.UIProcess


Namespace UIProcessQuickstarts_Store.WinUI
   
   Public Class congratulation
      Inherits WindowsFormView
      Private WithEvents returnButton As System.Windows.Forms.Button
      Private msgLabel As System.Windows.Forms.Label
      
        ' <summary>
        ' Required designer variable.
        ' </summary>
      Private components As System.ComponentModel.Container = Nothing
      
      
      Public Sub New()
         '
         ' Required for Windows Form Designer support
         '
         InitializeComponent()
      End Sub 'New
      
      
        ' <summary>
        ' Clean up any resources being used.
        ' </summary>
        Protected Overloads Overrides Sub Dispose(ByVal disposing As Boolean)
            If disposing Then
                If Not (components Is Nothing) Then
                    components.Dispose()
                End If
            End If
            MyBase.Dispose(disposing)
        End Sub 'Dispose

#Region "Windows Form Designer generated code"

        '/ <summary>
        '/ Required method for Designer support - do not modify
        '/ the contents of this method with the code editor.
        '/ </summary>
        Private Sub InitializeComponent()
            Me.msgLabel = New System.Windows.Forms.Label
            Me.returnButton = New System.Windows.Forms.Button
            Me.SuspendLayout()
            ' 
            ' msgLabel
            ' 
            Me.msgLabel.Location = New System.Drawing.Point(32, 24)
            Me.msgLabel.Name = "msgLabel"
            Me.msgLabel.Size = New System.Drawing.Size(232, 32)
            Me.msgLabel.TabIndex = 0
            Me.msgLabel.Text = "Congratulations. Your order has been sent sucessfully"
            Me.msgLabel.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
            ' 
            ' returnButton
            ' 
            Me.returnButton.Location = New System.Drawing.Point(96, 72)
            Me.returnButton.Name = "returnButton"
            Me.returnButton.Size = New System.Drawing.Size(104, 24)
            Me.returnButton.TabIndex = 1
            Me.returnButton.Text = "Return"
            ' 
            ' congratulation
            ' 
            Me.AutoScaleBaseSize = New System.Drawing.Size(5, 13)
            Me.ClientSize = New System.Drawing.Size(288, 109)
            Me.ControlBox = False
            Me.Controls.Add(returnButton)
            Me.Controls.Add(msgLabel)
            Me.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedDialog
            Me.MaximizeBox = False
            Me.Name = "congratulation"
            Me.StartPosition = System.Windows.Forms.FormStartPosition.CenterScreen
            Me.Text = "Congratulations"
            Me.ResumeLayout(False)
        End Sub 'InitializeComponent 
#End Region


        Private Sub returnButton_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles returnButton.Click
            StoreControllerHostedControl.ResumeShopping()
        End Sub 'returnButton_Click


        Private Sub congratulation_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles MyBase.Load
        End Sub 'congratulation_Load

#Region "UIPManager Plumbing"

        Private ReadOnly Property StoreControllerHostedControl() As StoreControllerHostedControl
            Get
                Return CType(Me.Controller, StoreControllerHostedControl)
            End Get
        End Property
#End Region
    End Class 'congratulation
End Namespace 'UIProcessQuickstarts_Store.WinUI